_includes/common/security.md

```diff
@@ -34,6 +34,12 @@ Almost every class that you create should have these permissions tweaked to some
 
 As a start, you can configure your application so that clients cannot create new classes on Parse. This is done by setting the key `allowClientClassCreation` to `false` in your ParseServer configuration.  See the project Readme for an overview of [Configuring your ParseServer](https://github.com/parse-community/parse-server#configuration).   Once restricted, classes may only be created from the Data Browser or with a the `masterKey`. This will prevent attackers from filling your database with unlimited, arbitrary new classes.
 
+### Enforcing Private Users
+
+*Available only on Parse Server starting 5.0.0*
```

Comment:
```suggestion
*Requires Parse Server 5.0.0+*
```

Reply:
We have some different ways of indicating this, so I'd go with the most compact one.